bench_vllm/setup.sh

```diff
@@ -44,9 +46,9 @@ install_vllm_cuda() {
         fi
         echo "Installing vllm for CUDA 11.8 with Python version: $PY_VERSION"
         # Download vllm for CUDA 11.8 and specified Python version
-        pip install https://github.com/vllm-project/vllm/releases/download/v0.2.2/vllm-0.2.2+cu118-"$PY_VERSION"-"$PY_VERSION"-manylinux1_x86_64.whl
-        pip install torch --upgrade --index-url https://download.pytorch.org/whl/cu118
-        pip install huggingface-cli
+        "$PYTHON_CMD" -m pip install https://github.com/vllm-project/vllm/releases/download/v0.2.2/vllm-0.2.2+cu118-"$PY_VERSION"-"$PY_VERSION"-manylinux1_x86_64.whl
+        "$PYTHON_CMD" -m pip install torch --upgrade --index-url https://download.pytorch.org/whl/cu118
+        "$PYTHON_CMD" -m pip install huggingface-cli
```

Comment:
can we add also here the version of the packages?

Reply:
Ah yes, I missed that here, lemme add those

Reply:
Done

Reply:
can you add also torch version?

Reply:
`torch --upgrade --index-url https://download.pytorch.org/whl/cu118`

This one is something I added from [vLLM documentation](https://docs.vllm.ai/en/latest/getting_started/installation.html), so I intentionally did not kept the version, just not to mess up with version issues.